frontend/src/components/AuthAmplify.tsx

```diff
@@ -13,13 +13,14 @@ type Props = BaseProps & {
 const AuthAmplify: React.FC<Props> = ({ socialProviders, children }) => {
   const { t } = useTranslation();
   const { signOut } = useAuthenticator();
+  const mistralEnabled: boolean = import.meta.env.VITE_APP_ENABLE_MISTRAL === 'true';
```

Comment:
Could you move to the top level of this module? The main reason is `enableMistral` is an env var, which means global variable.

Reply:
updated. please review again.